src/components/cluster/Cluster.tsx

```diff
@@ -255,10 +257,16 @@ export default class ClusterList extends Component<ClusterListProps, any> {
         return (
             <section className="global-configuration__component flex-1">
                 <div data-testid="cluster_and_env_header" className="flex left dc__content-space">
-                    <h2 className="form__title">{moduleBasedTitle}</h2>
+                    <FeatureTitleWithInfo
+                        title={moduleBasedTitle}
+                        renderDescriptionContent={() => `Manage your organization’s ${moduleBasedTitle.toLowerCase()}.`}
+                        docLink={ DOCUMENTATION.GLOBAL_CONFIG_CLUSTER}
+                        showInfoIconTippy
+                        additionalContainerClasses="mb-20"
+                    />
                     <button
                         type="button"
-                        className="flex cta h-32 lh-n fcb-5"
+                        className="flex cta h-32 lh-n fcb-5 mb-20"
```

Comment:
Why is this `mb-20` required here?

Reply:
It was sticking to the bottom form so added required margin